src/FixedDataTable.js

```diff
@@ -567,6 +567,7 @@ class FixedDataTable extends React.Component {
 
   componentDidMount() {
     this._reportContentHeight();
+    this._didControlledScroll(this.props);
```

Comment:
why do we need this?

Reply:
I assumed that didControlledScroll actually is supposed to trigger scroll, when scrollX and scrollY is present during mount I want it to be propagated to scrollbar as it is uncontrolled now.

Made changes according, now that I know `_didControlledScroll` is just triggering callbacks